WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -141,7 +136,7 @@ - (void)viewDidLoad
     UIBarButtonItem *backButton = [[UIBarButtonItem alloc] initWithTitle:[NSString string] style:UIBarButtonItemStylePlain target:nil action:nil];
     self.navigationItem.backBarButtonItem = backButton;
     
-    [self updateTabBarBadgeNumber];
+    [[WPTabBarController sharedInstance] updateNotificationBadgeVisibility];
```

Comment:
If this class is no longer involved with the Notifications badge handling (other than resetting it in the `viewWillAppear` method), perhaps this is not the right spot to place this call.

Maybe it should go in the WPTabBarController's own viewDidLoad method?